### PR TITLE
Added icheck-bootstrap@3.0.1

### PR DESCRIPTION
**Pull request by bantikyan**
# Pull request template for new library

Pull request for issue: N/A
Related issue(s): N/A

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): [https://github.com/bantikyan/icheck-bootstrap](https://github.com/bantikyan/icheck-bootstrap)
* Official website (optional, not the repository): [https://bantikyan.github.io/icheck-bootstrap/](https://bantikyan.github.io/icheck-bootstrap/)
* NPM package url (optional): [https://www.npmjs.com/package/icheck-bootstrap](https://www.npmjs.com/package/icheck-bootstrap)
* License and its reference: [MIT](https://github.com/bantikyan/icheck-bootstrap/blob/master/LICENSE)
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 3
  * Count of stars: 43
  * Count of forks: 7
* NPM download stats (optional):
  * Downloads in the last day: 
  * Downloads in the last week:
  * Downloads in the last month:

## Essential checklist

* [x] I'm the author of this library
  * [x] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR
* [x] The lib has notable popularity
  * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [ ] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [ ] Auto-update setup
* [ ] Auto-update target/source is valid.
* [ ] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [] The parent of the commit(s) in the PR is not older than 3 days.
* [x] Pull request is sent from a non-master branch with a meaningful name.
* [ ] Separate unrelated changes into different commits.
* [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by extend1994** [comment]
Thanks for the request! The library doesn't have enough popularity (200 stars/watchers on GitHub and 800 downloads/month on npm registry are good examples). You can take a look at our [issue rule](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#b-request-a-new-library) for more details :smile:

@PeterDaveHello Can we still host this lib?

File: ajax/libs/icheck-bootstrap/package.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "icheck-bootstrap",
+  "version": "3.0.1",
+  "description": "Pure css checkboxes and radio buttons for Twitter Bootstrap.",
+  "main": "icheck-bootstrap.css",
+  "files": [
+    "icheck-bootstrap.css",
+    "icheck-bootstrap.min.css"
+  ],
```
Comment (MattIPv4) [inline ajax/libs/icheck-bootstrap/package.json]:
```suggestion
```
`files` attr isn't needed.

File: ajax/libs/icheck-bootstrap/package.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  "bugs": {
+    "url": "https://github.com/bantikyan/icheck-bootstrap/issues"
+  },
```
Comment (MattIPv4) [inline ajax/libs/icheck-bootstrap/package.json]:
```suggestion
```

`bugs` attr isn't needed

File: ajax/libs/icheck-bootstrap/package.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  "npmName": "icheck-bootstrap",
+  "npmFileMap": [
+    {
+      "basePath": "/",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
+  "filename": "C:/Users/Mike/AppData/Local/Temp/tmp-8900ZsOqEyNDiobA/_config.yml"
```
Comment (MattIPv4) [inline ajax/libs/icheck-bootstrap/package.json]:
```suggestion
```

No idea what this is, use the `main` property (renamed to `filename`) above.

File: ajax/libs/icheck-bootstrap/package.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  "license": "MIT",
+  "npmName": "icheck-bootstrap",
+  "npmFileMap": [
+    {
+      "basePath": "/",
+      "files": [
+        "**/*"
+      ]
+    }
+  ],
```
Comment (MattIPv4) [inline ajax/libs/icheck-bootstrap/package.json]:
```suggestion
  ]
```

Invalid comma

File: ajax/libs/icheck-bootstrap/package.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    "radio",
+    "bootstrap",
+    "pure",
+    "css"
+  ],
+  "license": "MIT",
+  "npmName": "icheck-bootstrap",
+  "npmFileMap": [
+    {
+      "basePath": "/",
```
Comment (MattIPv4) [inline ajax/libs/icheck-bootstrap/package.json]:
```suggestion
      "basePath": "",
```

Base path should be blank for the root, not `/`

File: ajax/libs/icheck-bootstrap/package.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "icheck-bootstrap",
+  "description": "Pure css checkboxes and radio buttons for Twitter Bootstrap.",
+  "filename": "icheck-bootstrap.css",
```
Comment (MattIPv4) [inline ajax/libs/icheck-bootstrap/package.json]:
```suggestion
  "filename": "icheck-bootstrap.min.css",
```

A minified version exists, so we should prefer to suggest that.

File: ajax/libs/icheck-bootstrap/package.json [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+    "pure",
+    "css"
+  ],
+  "license": "MIT",
+  "npmName": "icheck-bootstrap",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "**/*"
```
Comment (MattIPv4) [inline ajax/libs/icheck-bootstrap/package.json]:
```suggestion
        "**/*.css"
```

We only need the css files, not license etc.

**Pull request merged**